packages/react-ui-screenshot-tests/gemini/input.js

```diff
@@ -175,3 +175,21 @@ gemini.suite('Input text styles reset', suite => {
     .setCaptureElements('#test-element')
     .capture('plain');
 });
+
+gemini.suite('Input error and focus', suite => {
+  suite
+    .before(renderStory('Input', 'Error and focus'))
+    .setCaptureElements('#test-element')
+    .capture('focused', (actions, find) => {
+      actions.click(find('label'));
```

Comment:
```suggestion
      actions.focus(find('input'));
```
Смотрелось бы лучше 🙂

Reply:
Пробовал, но работает нестабильно.

Reply:
![image](https://user-images.githubusercontent.com/4607770/61863872-e2f79480-aee9-11e9-9dd8-180aed399ab4.png)